### DISCUSSION TITLE
Proposal: Beta Tester (or similar) badge/award

### DISCUSSION THREAD

**Discussion by rozukke** [ppy/osu#26105] [Ideas]
With the launch of Lazer into the mainstream, the leaderboards and scores of the "beta" will finally, and somehow for the first time, be wiped. There has been a small but dedicated community playing the game full-time, so to speak, and they've built up a little history of their own within this in-development version, mostly supported by the fact that leaderboards were never actually reset. 
I think it would be nice for players who dedicated non-trivial amounts of time to playing Lazer to get a little nod of some sort to help distinguish them among the influx of people who will be playing Lazer seriously for the first time.
I do have a vested interest in this (since I've been playing Lazer as my main client since like 2019 or whenever skins were first implemented) but I do really love the community that has sprung up around the evolution that the client has undergone. 
I'm not exactly sure what the cutoff would be, the easiest way to tell would probably be the number of submitted scores or hours played? Feel free to chip in for any ideas regarding the whole proposal.

**Comment by peppy** [comment]
We're definitely considering something like this. Especially if we are resetting / deleting lazer scores until now, I'd like to give those people who did set scores some statistics in summary form at very least.

**Reply by Sanch-KK** [reply]
Should all the scores actually get wiped in the first place? Many people, including me, have switched to lazer because it just provides better experience than stable and have invested significant time into grinding for good scores, just like they would do before. I understand that there are many cases where such scores would break existing leaderboards or pp system, but at least scores with "generic" and safe mod combinations could be migrated, which would still mean alot for those who played lazer at least semi-seriously

**Reply by peppy** [reply]
scores may be migrated in the future but to keep things manageable for our small team we're not doing this initially.

we will keep an archive of scores set though.
